ql/src/main/resources/META-INF/services/org.apache.hadoop.hive.ql.secrets.SecretSource

```diff
@@ -0,0 +1 @@
+org.apache.hadoop.hive.ql.secrets.AWSSecretsManagerSecretSource
```

Comment:
this adds a hard dependency on the aws-sdk-bundle. Should we make this configuration based?